client/blocks/eligibility-warnings/hold-list.tsx

```diff
@@ -94,24 +65,82 @@ function getHoldMessages( translate: LocalizeProps[ 'translate' ] ) {
 	};
 }
 
+function getBlockingMessages( translate: LocalizeProps[ 'translate' ] ) {
+	return {
+		BLOCKED_ATOMIC_TRANSFER: {
+			message: translate(
+				'This site is not currently eligible to install themes and plugins. Please contact our support team for help.'
+			),
+			status: 'is-error',
+			contactUrl: localizeUrl( 'https://wordpress.com/help/contact' ),
+		},
+		TRANSFER_ALREADY_EXISTS: {
+			message: translate(
+				'Installation in progress. Just a minute! Please wait until the installation is finisehd, then try again.'
+			),
+			status: null,
+			contactUrl: null,
+		},
+		NO_JETPACK_SITES: {
+			message: translate( 'Try using a different site.' ),
+			status: 'is-error',
+			contactUrl: null,
+		},
+		NO_VIP_SITES: {
+			message: translate( 'Try using a different site.' ),
+			status: 'is-error',
+			contactUrl: null,
+		},
+		SITE_GRAYLISTED: {
+			message: translate(
+				"There's an ongoing site dispute. Contact us to review your site's standing and resolve the dispute."
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Contact us to review your site's standing and resolve the dispute.' )` *ES Score: 6*

Reply:
Merging the titles and messages like this is going to result in a number of places where we'll be waiting for localized strings.  Should we either:
* String freeze this PR & hold for translations
* Do something like: `message: hasTranslation( "There's an ongoing site dispute. Contact us to review your site's standing and resolve the dispute." ) ? translate( "There's an ongoing site dispute. Contact us to review your site's standing and resolve the dispute." : translate( 'Contact us to review your site's standing and resolve the dispute.' ),`

Reply:
I was thinking about shipping this one without waiting for translations since it's a way better experience than what we currently have, `hasTranslation` is just perfect - I'm going to update this PR right now.